README.md

```diff
@@ -90,6 +90,14 @@ Type: `Boolean`
 
 Default: `false`
 
+##### `options.unique`
```

Comment:
After sleeping on it, I think this option should probably be named `uniqueBy` to properly describe what it is doing. @contra thoughts?